<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommunicationSettingClass</class>
 <widget class="QWidget" name="CommunicationSettingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommunicationSetting</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget_AddContents">
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QListWidget
{
	color:#DBDBDB;
    background-color: rgb(50, 50, 50);
    border:1px solid back;
    border-top-left-radius:8;
    border-top-right-radius:8;
    outline:0px;
}
QListWidget::item
{
    background-color: rgb(50, 50, 50);    
}
QListWidget::item:hover
{
    background-color: rgb(65, 65, 65);
}
QListWidget::item::selected:active
{ 
   background-color: rgb(75, 75, 75);
}
QListWidget::item:selected
{
    background-color: rgb(75, 75, 75);
}</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_AddControl">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #262626;
border-bottom-left-radius:8;
border-bottom-right-radius:8;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="2">
           <widget class="MyLogoButton" name="btn_Reduce" native="true">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="MyLogoButton" name="btn_Add" native="true">
            <property name="minimumSize">
             <size>
              <width>75</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget_Communication"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyLogoButton</class>
   <extends>QWidget</extends>
   <header>UI/MyWidget/MyLogoButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
